### DISCUSSION TITLE
Responsive semantic tokens

### DISCUSSION THREAD

**Discussion by ptakpatryk** [chakra-ui/chakra-ui#8952] [Q&A] [answered]
Please let me know if I misunderstood the docs:
- https://next.chakra-ui.com/docs/theming/semantic-tokens#conditional-token
- https://panda-css.com/docs/concepts/conditional-styles

Am I able to write semantic tokens where the value changes for each breakpoint?
```ts
      spacing: {
        generous: {
          value: {
            base: '{spacing.3.5}',
            sm: '{spacing.7}',
            md: '{spacing.14}',
            lg: '{spacing.56}',
            _focus: '{spacing.56}',
            xl: '{spacing.56}',
            '2xl': '{spacing.56}'
          }
        }
      }
```

and then use it like this:
```ts
      <Box p='20px' mb="generous">
        Text here
      </Box>
```

It seems to be only using the `base` value, and none if `base` isn't specified. What am I missing?

**Comment by segunadebayo** [comment, marked as answer]
Breakpoint keys (`sm`, `md`) work as conditions on a semantic token. `_focus` is a different condition. Do not mix them in one object and expect every key to win.

```ts
generous: {
  value: { base: "{spacing.3.5}", sm: "{spacing.7}", md: "{spacing.14}" },
}
```

Then `mb="generous"`.
